### PR TITLE
Roundstart borgs aren't tater targets anymore.

### LINKED ISSUE #5831: (WEB REPORT BY: senaat1 REMOTE: 198.245.63.50:7777) It's possible to get cyborgs as traitor targets
Revision: 6dbab903e221204841d4c9a8072b44470c6e8df7 
Rolled traitor, got a cyborg as assassinate objective. Name displayed is (obviously) the character's name, not the borg's.

### PR DESCRIPTION
**Pull request by Intigracy**
fixes #5831

### CODE REVIEW

File: code/game/gamemodes/objective.dm [hunk 1]
```diff
@@ -29,8 +29,11 @@ datum/objective
 		//writepanic("[__FILE__].[__LINE__] ([src.type])([usr ? usr.ckey : ""])  \\proc/find_target() called tick#: [world.time]")
 		var/list/possible_targets = list()
 		for(var/datum/mind/possible_target in ticker.minds)
-			if(possible_target != owner && ishuman(possible_target.current) && (possible_target.current.z != map.zCentcomm) && (possible_target.current.stat != DEAD) && !(possible_target.assigned_role in list(bad_targets)))
```
Comment (clusterfack) [inline code/game/gamemodes/objective.dm]: possible_target.assigned_role in bad_targets would have sufficed here

Comment (Intigracy) [inline code/game/gamemodes/objective.dm]: Want me to change it?

Comment (clusterfack) [inline code/game/gamemodes/objective.dm]: yes

**Comment by Intigracy** [comment]
done

**Comment by Probe1** [comment]
:+1:

**Comment by IratePirate** [comment]
But it's fun to actually be a target.

**Comment by Intigracy** [comment]
I'll probably add in special objectives for targeting the AI / cyborgs, but the current ones inentionally have borgs/AIs/mommis count as dead.

**Pull request merged**